assets/airports/klas.json

```diff
@@ -463,18 +562,70 @@
                 "KLAS26R": []
             },
             "draw": [
-                ["DAG", "MISEN", "CLARR", "SKEBR", "CFBRW", "IPUMY", "BLD"],
-                ["TNP", "JOTNU", "GFS", "CLARR"]
+                ["BCE*", "HOLDM*", "AALAN", "CHOWW", "BLAID"],
+                ["PGA*", "HOLDM"]
+            ]
+        },
+        "CHOWW1": {
+            "icao": "CHOWW1",
+            "name": "Chow One",
+            "entryPoints": {
+                "STEWW": ["STEWW", ["RIGLL", "A240+"], ["AALAN", "A230-|A170+"]],
+                "GGAPP": ["GGAPP", "MNCIN", ["PHHIL", "A240+"], ["AALAN", "A230-|A170+"]],
+                "TYEGR": ["TYEGR", "WARKA", "DUUGG", ["HAANG", "A240+"], ["AALAN", "A230-|A170+"]]
+            },
+            "body": [["CHOWW", "A190-|S250"], "WOLPH"],
+            "rwy": {
+                "KLAS1L": [["PACKK", "A170+"], ["BEEPR", "A150+"], ["RFUSE", "A120+"], ["DNZIG", "A110+"], "JAIDE", ["TUUTH", "A100+|A80-|S220"], ["TRREY", "A70+|S210"]],
+                "KLAS1R": [["PACKK", "A170+"], ["BEEPR", "A150+"], ["RFUSE", "A120+"], ["DNZIG", "A110+"], "JAIDE", ["TUUTH", "A100+|A80-|S220"], ["^BUHLL", "A70|S210"], "#334"],
+                "KLAS19L": [["SACHL", "A100+"], ["FELAA", "A85+"], ["FOOOF", "A70+|S220"], "NNEON", ["^PPENN", "A60|S210"], "#263"],
+                "KLAS19R": [["SACHL", "A100+"], ["FELAA", "A85+"], ["FOOOF", "A70+|S220"], "NNEON", ["^PPENN", "A60|S210"], "#263"],
+                "KLAS26L": [["TATUU", "A105-|A95+"], ["KRLOZ", "A90-"], ["PRINO", "A80|S210"]],
+                "KLAS26R": [["TATUU", "A105-|A95+"], ["KRLOZ", "A90-"], ["FLYES", "A80|S210"]]
+            },
+            "draw": [
+                ["STEWW*", "RIGLL", "AALAN", "CHOWW", "WOLPH", "PACKK", "BEEPR", "RFUSE", "DNZIG", "JAIDE", "TUUTH", "TRREY"],
+                ["TUUTH", "BUHLL"],
+                ["GGAPP*", "MNCIN", "PHHIL", "AALAN"],
+                ["TYEGR*", "WARKA", "DUUGG", "HAANG", "AALAN"],
+                ["WOLPH", "SACHL", "FELAA", "FOOOF", "NNEON"],
+                ["WOLPH", "TATUU", "KRLOZ", "PRINO"],
+                ["KRLOZ", "FLYES"]
+            ]
+        },
+        "COKTL1": {
+            "icao": "COKTL1",
+            "name": "Cocktail One",
+            "entryPoints": {
+                "BASIC": ["BASIC", ["FLCHR", "A240+"], "TBRAN", ["ARYEL", "A210+"], ["COKTL", "A190-|A160+|S250"]],
+                "GIINN": ["GIINN", "TBRAN", ["ARYEL", "A210+"], ["COKTL", "A190-|A160+|S250"]]
```

Comment:
Although it should work fine, the the correct format for the restriction at COKTL is `A160+|A190-|S250`